examples/tutorial/jupyter/execution/pandas_on_dask/local/exercise_1.ipynb

```diff
@@ -0,0 +1,256 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p style=\"text-align:left;\">\n",
+    "        <h1>pandas on a multicore laptop\n",
+    "    <span style=\"float:right;\">\n",
+    "        Modin on a multicore laptop\n",
+    "    </span>\n",
+    "\n",
+    "<div>\n",
+    "<img align=\"left\" src=\"../../../img/pandas_multicore.png\"><img src=\"../../../img/modin_multicore.png\">\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Concept for exercise: setting Modin engine\n",
+    "\n",
+    "Modin uses Ray engine by default so no additional action is required to start to use it. Alternatively, if you need to use another engine, it should be specified either by setting the Modin config or by setting environment variable before the first `DataFrame` creation or data ingestion as it shown below. Also note, that full list of Modin configs and corresponding environment variables can be found in the [special section](https://modin.readthedocs.io/en/stable/flow/modin/config.html#modin-configs-list) of the Usage Guide in the Modin documentation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Modin engine can be specified either by config\n",
+    "import modin.config as cfg\n",
+    "cfg.Engine.put(\"dask\")\n",
+    "\n",
+    "# or environment variable setting\n",
+    "# import os\n",
+    "# os.environ[\"MODIN_ENGINE\"] = \"dask\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Concept for exercise: Dataframe constructor\n",
+    "\n",
+    "Often when playing around in Pandas, it is useful to create a DataFrame with the constructor. That is where we will start.\n",
+    "\n",
+    "```python\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**10, 2**5))\n",
+    "df = pd.DataFrame(frame_data)\n",
+    "```\n",
+    "\n",
+    "When creating a dataframe from a non-distributed object, it will take extra time to partition the data. When this is happening, you will see this message:\n",
+    "\n",
+    "```\n",
+    "UserWarning: Distributing <class 'numpy.ndarray'> object. This may take some time.\n",
+    "```\n",
+    "\n",
+    "**In a later exercise, we will discuss times when it is not possible to speed up the computation, even with multiprocessing or multithreading.**"
```

Comment:
```suggestion
```
There is no any use of multiprocessing or multithreading in a later exercise so this should be removed.